skore/src/skore/project.py

```diff
@@ -41,6 +41,7 @@ def __init__(
     @singledispatchmethod
     def put(self, key: str, value: Any, on_error: Literal["warn", "raise"] = "warn"):
         """Add a value to the Project.
+        If an item with the same key already exists, its value is replaced by the new one.
 
         If `on_error` is "raise", any error stops the execution. If `on_error`
```

Comment:
```suggestion
        """Add a value to the Project.
        
        If an item with the same key already exists, its value is replaced by the new one.
        If `on_error` is "raise", any error stops the execution. If `on_error`
```
That line belongs in the longer description